kani-compiler/src/codegen_cprover_gotoc/utils/float_utils.rs

```diff
@@ -112,6 +183,25 @@ const F64_U32_UPPER: [u8; 8] = [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF0, 0x41];
 const F64_U64_UPPER: [u8; 8] = [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF0, 0x43]; // 18446744073709551616.0
 const F64_U128_UPPER: [u8; 8] = [0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xF0, 0x47]; // 340282366920938463463374607431768211456.0
 
+const F128_U_LOWER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0xFF, 0xBF,
+];
+const F128_U8_UPPER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x07, 0x40,
+];
+const F128_U16_UPPER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0F, 0x40,
+];
+const F128_U32_UPPER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x1F, 0x40,
+];
+const F128_U64_UPPER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x3F, 0x40,
+];
+const F128_U128_UPPER: [u8; 16] = [
+    0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x7F, 0x40,
+];
+
 /// upper is the smallest `f32` that after truncation is strictly larger than i<N>::MAX or u<N>::MAX
```

Comment:
Could you change this comment to make it clear that it's talking about `get_bounds_f32`? I found it confusing to have it above `get_bounds_f16`, since the (upper, lower) there aren't 32 bit floats.
I'm fine with however you do this--add a line to the comment saying this is an example for `f32` that generalizes to all the functions, changing the example to talk about `f16`s instead, moving `get_bounds_f32` above `get_bounds_f16`, etc.

Reply:
Sure, I'll fix that.